docs/functions/Bundles/Start-UPloadVCFBundle.md

```diff
@@ -0,0 +1,38 @@
+# Start-UploadVCFBundle
+
+### Synopsis
+The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
+
+### Syntax
+```
+Start-UploadVCFBundle -json <path to json file>
+```
+
+### Description
```

Comment:
Suggest expanding the description to explain that you first need to SCP the files to SDDC Manager as this implies that the function is doing the upload

Reply:
Fixed